kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RuntimeDataServiceIntegrationTest.java

```diff
@@ -422,6 +422,7 @@ public void testGetProcessInstancesByContainerSortedByName() throws Exception {
                     assertEquals(PROCESS_ID_SIGNAL_PROCESS, instances.get(i).getProcessId());
                 }
             }
+            assertEquals(Integer.valueOf(5), queryClient.countProcessInstancesByContainerId(CONTAINER_ID, null));
```

Comment:
Expected value is Long instead of Integer
```suggestion
            assertEquals(Long.valueOf(5), queryClient.countProcessInstancesByContainerId(CONTAINER_ID, null));
```